code/game/objects/items/climbingrope.dm

```diff
@@ -69,7 +69,7 @@
 	if(do_after(user, final_climb_time, interacting_with))
 		user.forceMove(interacting_with)
 		uses--
-		user.mind?.adjust_experience(/datum/skill/athletics, 50) //get some experience for our trouble, especially since this costs us a climbing rope use
+		user.mind?.adjust_experience(/datum/skill/athletics, round(50/(fitness_level || 1), 1)) //get some experience for our trouble, especially since this costs us a climbing rope use
```

Comment:
Might want to make the hardcoded values for xp rewards a define instead of hardcoding `50` and do the same with ladders.

Reply:
I did this.